frontend/src/components/object/DeleteObjectButton.vue

```diff
@@ -33,13 +35,13 @@ const confirmDelete = () => {
   if (props.ids.length) {
     const msgContext = props.ids.length > 1 ? `the selected ${props.ids.length} files` : 'this file';
```

Comment:
feels like the confirm message should specify if its a version or an object being deleted. you could do:
```js
const item = props.versionId ? 'version' : 'object';  
const msgContext = props.ids.length > 1 ? `the selected ${props.ids.length} ${item}s` : 'this ${item}';
header: `Delete ${props.ids.length > 1 ? item + 's' : item }`,
...
toast.error('Error deleting one or more ' + item + 's');
```
i suspect when we look at doing hard/soft deletes this will need to be re-visited